views/tables/ParticipantsTable/index.jsx

```diff
@@ -50,6 +50,11 @@ const ParticipantsTable = (props) => {
       label: 'Last synced',
       sortable: false,
     },
+    {
+      dataProperty: 'Active',
+      label: 'Status',
+      sortable: false,
```

Comment:
Should this be sortable eventually? I don't think it's part of this ticket if so, just curious

Reply:
good catch